microprofile-nosql/pom.xml

```diff
@@ -0,0 +1,159 @@
+        <artifactId>fabric8-maven-plugin</artifactId>
+        <version>3.5.41</version>
+        <executions>
+          <execution>
+            <goals>
+              <goal>resource</goal>
+            </goals>
+          </execution>
+          <execution>
+            <id>execution1</id>
+                        <configuration>
+              <!-- Standard d-m-p configuration which defines how images are build, i.e. how the docker.tar is created -->
+              <images>
+                <image>
+                  <name>mongo</name>
+                  <!-- "alias" is used to correlate to the containers in the pod spec -->
+                  <alias>mymongo</alias>
+                </image>
+              </images>
+              <!-- resources to be created -->
+              <resources>
+                <!-- Labels that are applied to all created objects -->
+                <labels>
+                  <group>quickstarts</group>
+                </labels>
+                <!-- Definition of the ReplicationController / ReplicaSet. Any better name than "containers" ? -->
+                <deployment>
+                  <!-- Name of the replication controller, which will have a sane default (container alisa, mvn coords, ..) -->
+                  <name>${project.artifactId}</name>
+                  <!-- Replica count-->
+                  <replicas>1</replicas>
+                  <!-- Container to include in the POD template. By default all with a "build" section in the
+                       "images" configuration will be add to the POD. However, they can be configured separately, too.
+                      -->
+                  <containers>
+                    <container>
+                      <!-- Alias name correlating with the same named "image" configuration above. Can be ommitted
+                           if there is only a single image added -->
+                      <alias>camel-service</alias>
+                      <ports>
+                        <!-- Ports to expose in the pod specs -->
+                        <port>8778</port>
+                      </ports>
+                      <mounts>
+                        <scratch>/var/scratch</scratch>
+                      </mounts>
+                    </container>
+                  </containers>
+                  <!-- Volumes used in the replicaSet -->
+                  <volumes>
+                    <volume>
+                      <name>scratch</name>
+                      <type>emptyDir</type>
+                    </volume>
+                  </volumes>
+                </deployment>
+                <!-- Dedicated section for (multiple) services to define -->
+                <services>
+                  <service>
+                    <name>${project.artifactId}</name>
+                    <headless>true</headless>
+                  </service>
+                </services>
+              </resources>
+            </configuration>
+          </execution>
```

Comment:
I don't quite understand why this entire `<execution>` is needed. Seems to me your usage of FMP is unnecessarily complicated.